subsys/net/lib/nrf70_fw_ext/ext_fw_load.c

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @brief File containing FW load functions for Zephyr.
+ */
+#include <stdlib.h>
+#include <string.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#if defined(CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP) && defined(CONFIG_NORDIC_QSPI_NOR)
+#include <zephyr/drivers/flash/nrf_qspi_nor.h>
+#endif /* CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP */
+
+#include <zephyr/logging/log.h>
+#if defined(CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP) && defined(CONFIG_NORDIC_QSPI_NOR)
+/* For NRF QSPI NOR special handling is needed for this file as all RODATA of
+ * this file is stored in external flash, so, any use of RODATA has to be protected
+ * by disabling XIP and enabling it again after use. This means no LOG_* macros
+ * (buffered) or buffered printk can be used in this file, else it will crash.
+ */
+LOG_MODULE_DECLARE(wifi_nrf, LOG_LEVEL_NONE);
+#else
+LOG_MODULE_DECLARE(wifi_nrf, CONFIG_WIFI_NRF70_LOG_LEVEL);
+#endif /* CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP */
+
+#include <fmac_main.h>
+
+#ifdef CONFIG_NRF_WIFI_PATCHES_EXT_FLASH_XIP
+static const char nrf70_fw_patch[] = {
+	#include <nrf70_fw_patch/nrf70.bin.inc>
```

Comment:
where is this file generated?

Reply:
https://github.com/nrfconnect/sdk-zephyr/pull/2062/commits/30100c70239da1201849ec673fde8228a5cbdd98#diff-9171118f1e64973f46825a6d56db0770019da60848725b7256a131d045a3a706R175